### ISSUE TITLE
Feature Request: save exact command-line invocation of bpipe in commandlog.txt

### ISSUE THREAD

**Issue by gdevenyi** [ssadedin/bpipe#138]
Sometimes I want to re-run an interrupted pipeline but I'm not sure exactly how I invoked it.

If commandlog.txt contained that in the header it would be useful.

```
#bpipe invocation command:  bpipe run ../minc-bpipe-library/pipeline.bpipe ../input/*
```

**Comment by ssadedin** [comment]
Just noting that I looked into this but it's unfortunately impossible to do because the shell interprets the wildcards before Bpipe ever sees them. So the best we can do is to save the command line with the wildcards expanded. That shouldn't be too hard (the same is already written to the Bpipe history). If you're not familiar with "bpipe history" then it may get you part way there. However the history doesn't definitively link the commands executed to the invocation, so it would be good to implement this.

**Comment by gdevenyi** [comment]
Right, of course the shell will expand any wildcards, that's fine.

I'm familiar with "bpipe history", but I'm also a big fan of having a single logfile that concisely describes what was done to the files. commandlog.txt is very close to this already.

In fact, I'm pretty sure I can reproduce my pipelines by running the shell on commmand.txt. not sure that was intentional, but it is very important for scientific reproducibility.
